README.rst

```diff
@@ -165,11 +377,18 @@ Future
 
 Open edX "Hawthorn" will provide a plug-in architecture. This will hopefully simplify Figures installation.
 
-Contributing
-------------
+How to Contribute
+-----------------
+
 
 TODO: Add details here or separate `CONTRIBUTING` file to the root of the repo
```

Comment:
Will this be a separate PR?

Reply:
Yes, this will be a separate PR. Created trello card for it: https://trello.com/c/gc5PHhsS/444-1-add-contributing-to-figures